### DISCUSSION TITLE
Questions about running on android

### DISCUSSION THREAD

**Discussion by Cacti2432** [ORelio/Minecraft-Console-Client#2739] [Q&A]
Does installing Ubuntu do anything to my operating system? (I am very new to this)
If I follow the steps wrongly will there be any permanent damage to my device?

**Comment by milutinke** [comment]
> Does installing Ubuntu do anything to my operating system? (I am very new to this)
> If I follow the steps wrongly will there be any permanent damage to my device?

Nope, because it's all wrapped  by Termux.
If you make a mistake, you can always correct it or start from the beginning, it will not affect your main operating system.

Make sure to install the Termux from F-Droid and not from the Play Store, and also turn off battery optimization for it so it can run in the background.
